src/Item/Video/VideoItemValidator.php

```diff
@@ -65,11 +65,16 @@ public function validateThumbnailLoc($loc)
     /**
      * @param $title
      *
-     * @return bool
+     * @return string|false
```

Comment:
I think the return annotation should be type hint.

That is, it should be `string|bool`.